packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -14,10 +14,14 @@ interface UseFeedLayout {
   shouldUseFeedLayoutV1: boolean;
 }
 
+export const FeedLayoutV1FeedPages = new Set(
+  Object.values(SharedFeedPage).filter(
+    (feedPage) => feedPage !== SharedFeedPage.Search,
```

Comment:
Search and Squad should be excluded right?
@nensidosari

Reply:
Squad feed and search should not be part of the v1 feeds. See this thread: https://dailydotdev.slack.com/archives/C061T644XV3/p1705320332454659?thread_ts=1705316572.801329&cid=C061T644XV3

Reply:
Yes yes, why you only see Search mentioned there @rebelchris  is because Squad is not part of SharedFeedPage.

Reply:
Ah bit confusing to read then :)